docs/Makefile

```diff
@@ -29,16 +29,28 @@ run_animate_demos:
 	cd . \
 
 run_goalie_demos:
+	@SKIP_DEMOS="time_partition.py gray_scott.py gray_scott_split.py"; \
```

Comment:
What does the `@` mean here? I haven't seen this notation before and as far as I can tell it is redundant.

Reply:
It just means that whatever is after `@` doesn't get echoed to console. Minor :) I removed the `@` in https://github.com/mesh-adaptation/docs/pull/92/commits/7906cf4abfaafd8a205170598c4f63a44450aefb

Reply:
Ah yes I did know that. Thanks!